build.sbt

```diff
@@ -18,13 +18,13 @@ lazy val publishSettings = Seq(
   organizationHomepage := Some(url("https://www.apache.org")),
   sonatypeCredentialHost := apacheBaseRepo,
   sonatypeProfileName := "org.apache.pekko",
-  startYear := Some(2022),
+  startYear := Some(2023),
   developers := List(
     Developer(
-      "pekko-contributors",
-      "Apache Pekko Contributors",
+      "pekko-sbt-paradox-contributors",
+      "Apache Pekko Sbt Paradox Contributors",
```

Comment:
I don't mind but do we really have to make this distinction everywhere?

Reply:
* it's what we've been doing in other repos
* an example is that a contributor could contribute to this repo but not to the main pekko one - so the link would not acknowledge their work

Reply:
yes - only the link technically needs to be different but the point is that we sort of have this norm in the other repos too (being specific for all the developer params)